src/theme/typography.js

```diff
@@ -0,0 +1,58 @@
+const typography = {
+  // TODO: Add font.
```

Comment:
We're still waiting on the font right?

Reply:
Yeah. It can be updated here once that is done.